lambdas/service/app.py

```diff
@@ -416,19 +416,27 @@ def openapi():
 }
 
 fast_health_keys = {
-    **{
-        prop.key: format_description(prop.description)
-        for prop in HealthController.fast_properties['service']
-    },
-    **health_up_key
+    p: d for p, d in sorted(
+        {
+            **{
+                prop.key: format_description(prop.description)
+                for prop in HealthController.fast_properties['service']
+            },
+            **health_up_key
+        }.items(),
+        key=lambda key: key[0])
```

Comment:
You are sorting an iterable of (k,v) tuples, and the keys (k) are unique, so the values (v) are never needed to break a tie. Since tuples are directly comparable, the default key should work. Also, to call the argument to a key function "key" is misleading. And look at `itemgetter` from `operators`. But you won't need it.